ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -173,6 +187,32 @@ class TabBarItemView: UIControl {
         }
     }
 
+    @objc private func isUnreadValueDidChange() {
+        isUnreadDotVisible = item.isUnreadDotVisible
+        updateUnreadDot()
+        updateAccessibilityLabel()
+        setNeedsLayout()
+    }
+
+    private var isUnreadDotVisible: Bool = false {
+        didSet {
+            if oldValue != isUnreadDotVisible {
```

Comment:
according to your other comment, if there is already a badgevalue that means we should completely igore the new value of isUnreadDotvisible?

Reply:
Eh, this feels like a safety release valve in case of weird behavior from the consumer. I'd rather keep this logic the same just in case they decide to remove a badge but not change the read/unread state.